draft-age-masque-connect-ip.md

```diff
@@ -137,6 +137,13 @@ routes to the proxy for network-to-network routing.
 
 ## Limiting Request Scope {#scope}
 
+Unlike CONNECT-UDP requests, which require specifying a target host, CONNECT-IP
+requests can allow endpoints to send arbitrary IP packets to any host.
+The client can choose to restrict a given request to a specific host or IP
+protocol by adding parameters to its request. The server uses these parameters
```

Comment:
This doesn't quite capture why this is useful. Perhaps something along the lines of "When the server knows that a request is scoped to a target host, it can leverage this information to optimize its resource allocation; for example, the server can assign the same public IP address to two CONNECT-IP requests that are scoped to different hosts."

Reply:
I like that! Updated with minor edits to your suggestion.